pkgs/applications/audio/vcv-rack/default.nix

```diff
@@ -7,10 +7,8 @@ let
   # Others are downloaded with `make deps`. Due to previous issues with the
   # `glfw` submodule (see above) and because we can not access the network when
   # building in a sandbox, we fetch the dependency source manually.
-  pfft-source = fetchFromBitbucket {
-    owner = "jpommier";
-    repo = "pffft";
-    rev = "74d7261be17cf659d5930d4830609406bd7553e3";
+  pfft-source = fetchzip {
+    url = "https://vcvrack.com/downloads/dep/pffft.zip";
```

Comment:
Can you upload it to archive.org?

Reply:
Good thought! Here it is: https://web.archive.org/web/20210103085010/https://vcvrack.com/downloads/dep/pffft.zip

Please note that the archive is still available at https://bitbucket.org/jpommier/pffft/get/74d7261be17cf659d5930d4830609406bd7553e3.tar.gz for the time being as well. I think we're covered.